.github/workflows/2_bump_versions.yml

```diff
@@ -0,0 +1,62 @@
+name: Bump versions
+
+on:
+  workflow_dispatch:
+    inputs:
+      npm_channel:
+        description: 'Prisma CLI NPM Channel'
+        required: true
+      version:
+        description: 'New Prisma CLI version in that channel'
+        required: true
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  ENVIRONMENT: ${{ secrets.ENVIRONMENT }}
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: Install Semver # Needed to use semver in scripts
+        run: npm install
+      - name: Setup Git
+        run: |
+          git config --global user.email "prismabots@gmail.com"
+          git config --global user.name "Prismo"
+      - name: Update versions in scripts folder 
+        run: |
+          echo "${{ github.event.inputs.version }}" >scripts/versions/prisma_${{ github.event.inputs.npm_channel }}
+      - name: Update next extension version in scripts folder
+        id: update # Used to access the calculated next extension version in later steps
+        run: node scripts/bump_extension_version_in_scripts.js ${{ github.event.inputs.npm_channel }}
+      - name: Push to master
+        run: |
+          git add -A .
+          git commit -m "bump Prisma CLI ${{ github.event.inputs.npm_channel }} version to ${{ github.event.inputs.version }} and extension version to ${{ steps.update.next_extension_version }} in scripts."
+          git pull --rebase
+          git push
+      - name: Switch branch 
+        id: setup_branch
+        run: npm run setup:branch ${{ github.event.inputs.npm_channel }}
```

Comment:
What does this really do? I see a shell script that then uses a JS script...

Reply:
The JS script gets the name of current branch name, even if it does not exist yet.
The shell script then checks if the branch already exists in git and checks out to that branch.

Reply:
Why do we need to do that, this way? Do we dynamically create a branch in some cases?

Reply:
yes, in case there is a patch from Prisma CLI for example, we check out to e.g. 2.5.x, but this branch does not necessarily exist yet. The git commands for checking out an existing branch and a newly created one differ, so we have to do it this way.